### PR TITLE
FirwareOffer layout fix

### LINKED ISSUE #6864: Firmware installation modal not properly aligned when using BTC only fw
**Describe the bug**
Firmware installation modal not properly aligned when using BTC only firmware.
I was not able to reproduce issue with regular firmware. 
Follow up of https://github.com/trezor/trezor-suite/issues/4261

**Steps to reproduce:**
1. Install BTC only firmware
2. Navigate to Settings/device section and start firmware update
3. Observe text alignment in modal.

**Info:**
 - Suite version: desktop 22.11.1 (73ccf723353e0575a64f58b2f5775d20f072311f)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.11.1 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.5.3 Bitcoin-only (revision 2f03ace311584988d5aeab58fd1acf24ef54711a)
 - Transport: bridge 2.0.32

**Screenshots:**
image

### PR DESCRIPTION
**Pull request by komret**
## Description
- removes linebreak from firmware version
- changes heading color
- component refactoring

## Related Issue

Resolve #6864

## Screenshots:
before:
Screenshot 2022-11-30 at 13 50 41
after:
Screenshot 2022-11-30 at 13 36 25

### CODE REVIEW

File: packages/suite/src/components/firmware/FirmwareOffer.tsx [hunk 1]
```diff
@@ -114,81 +100,64 @@ const FirmwareOffer = ({ device, customFirmware, targetFirmwareType }: Props) =>
...
-                            </Version>
-                        </Tooltip>
-                    ) : (
-                        <Version new>
-                            {formattedNextFirmwareType}
-                            {newVersion}
-                        </Version>
-                    )}
-                </VersionWrapper>
+                                                View all
```
Comment (dahaca) [inline packages/suite/src/components/firmware/FirmwareOffer.tsx]: Could be translated, right?

File: packages/suite/src/components/firmware/FirmwareOffer.tsx [hunk 2]
```diff
@@ -114,81 +100,64 @@ const FirmwareOffer = ({ device, customFirmware, targetFirmwareType }: Props) =>
...
-                                                href={
-                                                    parsedChangelog.notes
-                                                        ? parsedChangelog.notes
-                                                        : changelogUrl
-                                                }
+                {parsedChangelog ? (
+                    <Tooltip
+                        rich
+                        dashed
+                        content={
```
Comment (dahaca) [inline packages/suite/src/components/firmware/FirmwareOffer.tsx]: Perhaps the content could be taken out for a cleaner render?

File: packages/suite/src/components/firmware/FirmwareOffer.tsx (hunk 1 above)
Comment (komret) [reply packages/suite/src/components/firmware/FirmwareOffer.tsx]: The changelog is in English anyway, but yeah... `Translation` added in 05d94686197da26dd84461fc9a927226b60b5cc2.

File: packages/suite/src/components/firmware/FirmwareOffer.tsx (hunk 2 above)
Comment (komret) [reply packages/suite/src/components/firmware/FirmwareOffer.tsx]: Yes, I turned it into a component in 05d94686197da26dd84461fc9a927226b60b5cc2.

**Review by dahaca — APPROVED** [review]

**Pull request merged**